.travis.yml

```
@@ -20,8 +23,12 @@ jobs:
       python: 3.9
       script: export PYTHONPATH="src"; python -m unittest src/test_com2ann.py
 
-    - name: "Run tests on Python 3.10 (beta)"
-      python: 3.10-dev
+    - name: "Run tests on Python 3.10.2"
+      python: 3.10.2
```

Actually I think it may be better to try the latest 3.10
```suggestion
      python: 3.10
```

---

"3.10" MUST be quoted in YAML... https://dev.to/hugovk/the-python-3-1-problem-85g